source/developers/experience-builder.rst

```diff
@@ -418,7 +418,7 @@ Examples
 
 .. code-block:: text
 
-   <@crafter.body_bottom iceProps="{ scrollElement: '#mainWrapper' }" />
+   <@crafter.body_bottom props="{ scrollElement: '#mainWrapper' }" />
```

Comment:
It's `props` if you use `initExperienceBuilder` directly, but `iceProps` for `body_bottom`.